app/views/activities/_feed_entry.html.slim

```diff
@@ -1,10 +1,14 @@
 li class="activity #{activity.kind}"
   = link_to activity.source_url
-    = image_tag(activity.img_url || '/images/screenshot-missing.png')
-  = content_tag :h4, link_to(activity.title, activity.source_url, class: 'title')
+  	- if activity.img_url.present?
```

Comment:
Yes please. As a rule of thumb, mixing tabs and spaces in a code-base means trouble. Ruby has settled on 2 spaces for indentation (as did Python, I think), JS is on 4 spaces and so every other language has their community best practices